### DISCUSSION TITLE
OpenSeesの追加の提案

### DISCUSSION THREAD

**Discussion by tkoyama010** [FEABeRjp/FEABeR#28] [Ideas]
建築系の解析ソフト[OpenSees](https://github.com/OpenSees/OpenSees)の追加を提案します。

**Comment by tkoyama010** [comment]
Pythonのインターフェスがあるようです。
https://openseespydoc.readthedocs.io/en/latest/

**Reply by mmer547** [reply]
気になってたソフトウェアなのでいいと思います。しばらく触れそうにないですけど……
